### PR TITLE
Update spec version

### PR DESCRIPTION
**Pull request by sreevatsanraman**

### CODE REVIEW

File: core-plugins/widgets/SnapshotParquet-batchsource.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "metadata": {
-    "spec-version": "1.3"
+    "spec-version": "1.5"
   },
   "display-name": "Avro Parquet Dataset",
```
Comment (bdmogal) [inline core-plugins/widgets/SnapshotParquet-batchsource.json]: this seems wrong? - Parquet Snapshot Dataset?

File: core-plugins/widgets/SnapshotParquet-batchsink.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
 {
   "metadata": {
-    "spec-version": "1.3"
+    "spec-version": "1.5"
   },
   "display-name": "Avro Parquet Dataset",
```
Comment (bdmogal) [inline core-plugins/widgets/SnapshotParquet-batchsink.json]: ditto

Comment (bdmogal) [reply core-plugins/widgets/SnapshotParquet-batchsink.json]: Parquet Snapshot Dataset?

**Review by bdmogal — COMMENTED** [review]
couple minor comments. rest LGTM.

**Comment by sreevatsanraman** [comment]
Fixed comments. merging.

**Pull request merged**